sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -4,15 +4,23 @@
 
 ### Features Added
 
+- Added support for service API version, `2024-03-01-preview`.
+- Added two new properties to `EmbeddingOptions`:
+  - `dimensions`, which is only supported in models `text-embedding-3-*` and above.
```

Comment:
What is the second property added to `EmbeddingOptions`?

Reply:
It is `encodeing_format`.    
```
    @JsonProperty(value = "encoding_format")
    private EmbeddingEncodingFormat encodingFormat;
```